.github/workflows/python-tests.yml

```diff
@@ -33,4 +33,4 @@ jobs:
     - name: Test with pytest
       run: |
         pip install -r requirements-dev.txt
-        pytest tests
+        pytest tests/unit
```

Comment:
Not only unit tests should be run, the passed flag should mean all tests are passed